paddle/fluid/framework/ir/conv_bn_fuse_pass.cc

```diff
@@ -135,6 +64,13 @@ void recompute_bias_and_weights(const Scope* scope,
                                 const ir::Node& bn_variance,      //
                                 LoDTensor* eltwise_y_in_tensor,   //
                                 float epsilon) {
+  using EigenVectorArrayMap =
+      Eigen::Map<Eigen::Array<float, Eigen::Dynamic, 1>>;
+  using ConstEigenVectorArrayMap =
+      Eigen::Map<const Eigen::Array<float, Eigen::Dynamic, 1>>;
+  using EigenMatrixArrayMap = Eigen::Map<
+      Eigen::Array<float, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>>;
+
```

Comment:
PaddlePaddle has functions for wrapping Tensor objects into Eigen objects. They are defined in `paddle/fluid/framework/eigen.h`. Do you think they are useful for you?

Reply:
@sfraczek Is `paddle/fluid/framework/eigen.h` useful for you?

Reply:
I don't know how I can use it to my advantage. I tried to experiment EigenMatrix but I can't get it to broadcast. I think I need the array type not vector nor matrix.

Reply:
@luotao1, could you please accept the PR now? I think I can't use eigen.h in this case.